modules/oe_theme_content_news/oe_theme_content_news.post_update.php

```diff
@@ -83,3 +84,22 @@ function oe_theme_content_news_post_update_00005(): void {
 function oe_theme_content_news_post_update_00006(): void {
   \Drupal::service('module_installer')->install(['oe_theme_content_entity_contact']);
 }
+
+/**
+ * Create the 'full' entity view display on the news CT.
+ */
+function oe_theme_content_news_post_update_00007(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_news') . '/config/post_updates/00007_create_full_view_display');
+
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $config = $storage->read('core.entity_view_display.node.oe_news.full');
+  // We are creating the config which means that we are also shipping
+  // it in the config/install folder so we want to make sure it gets the hash
+  // so Drupal treats it as a shipped config. This means that it gets exposed
+  // to be translated via the locale system as well.
+  $config['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config));
```

Comment:
Since this comes from a post update directory, can't we just add it to the YAML itself?